### PR TITLE
Feat: Wishlist added

### PR DESCRIPTION
**Pull request by yashpatel1797**
Wishlist page added
[live link](https://confident-wozniak-7f4f57.netlify.app/page/wishlist.html)

### CODE REVIEW

**Review by kotesh-arya — APPROVED** [review]
Using well defined alt text for images is one of the best practices which improves USER EXPERIENCE & ACCESSIBILITY

File: page/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        </a>
+      </span>
+    </nav>
+    <h3 class="page-title">My Wishlist</h3>
+    <div class="grid-4-column wishlist">
+      <div class="card-vertical">
+        <div class="image-container">
+          <img
+            src="https://m.media-amazon.com/images/I/51vgy3LMz6L._AC_UY327_FMwebp_QL65_.jpg"
+            alt=""
```
Comment (sharmayatendra) [inline page/wishlist.html]: you can use proper alt attribute

**Review by sharmayatendra — COMMENTED** [review]
can make suggested changes else looks fine to merge

**Comment by agrawalshanu708** [comment]
Try to give different font size for card description, card tag,card title..and else is looking good

**Pull request merged**